src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java

```diff
@@ -347,8 +347,14 @@ public void clickSuEliminaPec() {
     public String waitLoadPopUpElimina() {
         By titlePopUp = By.id("dialog-title");
         By subTitlePopUp = By.id("dialog-description");
-        this.getWebDriverWait(10).withMessage("Non è stato caricato il titolo del pop-up").until(ExpectedConditions.visibilityOfElementLocated(titlePopUp));
-        this.getWebDriverWait(10).withMessage("Non è stato caricato il sottotitolo del pop-up").until(ExpectedConditions.visibilityOfElementLocated(subTitlePopUp));
+        By cancelButton = By.id("buttonAnnulla");
```

Comment:
This one is already present, use it and do the same for the confirmButton
https://github.com/pagopa/pn-frontend-e2e/blob/10524309fbd27d6c7d78103d86d876080fe07053/src/main/java/it/pn/frontend/e2e/common/RecapitiDestinatarioPage.java#L65-L66